cmd/resumic/tests/validator_test.go

```diff
@@ -0,0 +1,58 @@
+package tests
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/resumic/schema/cmd/resumic/validate"
+)
+
+//TestValidatorValidExamples used to test valid JSON passed in ValidateJSON
+func TestValidatorValidExamples(t *testing.T) {
+	files := run("../../../examples/valid")
+	for _, file := range files {
+		print("In ")
+		print(file)
+		println(":")
```

Comment:
Not sure I like the print statement.
Any reason, why you went with print instead of:
`fmt.Println("In %s:", file)`

Reply:
no reason as such was just for testing locally :p

Reply:
Sidenote: fmt.Print* is preferred in production code.